src/front/spv/mod.rs

```diff
@@ -1570,23 +1566,87 @@ impl<I: Iterator<Item = u32>> Parser<I> {
         let function_id = self.next()?;
         let (name, left) = self.next_string(inst.wc - 3)?;
         let ep = EntryPoint {
-            exec_model,
+            stage: match exec_model {
+                spirv::ExecutionModel::Vertex => crate::ShaderStage::Vertex,
+                spirv::ExecutionModel::Fragment => crate::ShaderStage::Fragment {
+                    depth_replacing: None,
+                    forced_early_depth_test: None,
+                    conservative_depth_test: None,
+                },
+                spirv::ExecutionModel::GLCompute => crate::ShaderStage::Compute {
+                    local_size: (8, 1, 1),
```

Comment:
why are we using this local size?

Reply:
No reason. I was simply not implementing the functionality for all the frontends, only fixing compilation errors. 

This PR only adds it to the IR. It is up to the individual frontend implementors to provide the correct numbers.

Alternative would be using Option but I feel like that would be pushing implementation detail to frontends.

Reply:
if there isn't a sensible value to put, then let's use a non-sensible one, like `[1; 3]` or `[0; 3]`

Reply:
Done